R/kpi_format.R

```diff
@@ -1,7 +1,40 @@
 #### Prepare a function to format the output (lists) from the function kpi
 
+### for the moment the function would work only if the risk is present, otherwise the colors would not make sense (?)
```

Comment:
agree... longer term, maybe we could add something like ggplots colours of continuous variables so that extreme values stand out...